tests/integration/any_charm.py

```diff
@@ -25,10 +28,8 @@ def update_ingress(self, ingress_config):
     @staticmethod
     def start_server(port: int = 80):
         """Start an HTTP server daemon."""
-        www_dir = pathlib.Path("/tmp/www")
-        www_dir.mkdir(exist_ok=True)
-        ok_file = www_dir / "ok"
-        ok_file.write_text("ok")
+        www_dir = tempfile.mkdtemp()
+        # We create a pid file to avoid concurrent executions of the http server
         pid_file = pathlib.Path("/tmp/any.pid")
```

Comment:
I realise this is merged, should this also be changed to use the `tempfile` module?